### DISCUSSION TITLE
Is there a way to deactivate user click in chart?

### DISCUSSION THREAD

**Discussion by JosephKorel** [recharts/recharts#4018] [Q&A] [answered]
Hi, from the website pie chart example, if i click in a slice, it gets selected, like this 
![image](https://github.com/recharts/recharts/assets/96602536/5962f1f8-1f16-4433-8538-d9bcf3996047)
Is there a way to deactivate that?

**Comment by HHongSeungWoo** [comment]
The easiest way is to use CSS. You can set the `pointer-events: none`

**Reply by JosephKorel** [reply]
I thought about that, but i need hover event :/

**Reply by HHongSeungWoo** [reply, marked as answer]
`.recharts-wrapper * {
    outline: none;
}` how about this?
